pkg/monitor/datapath_trace.go

```diff
@@ -108,16 +117,25 @@ func (n *TraceNotify) traceSummary() string {
 	}
 }
 
+// OriginalIP returns the original source IP if reverse NAT was performed on
+// the flow
+func (n *TraceNotify) OriginalIP() net.IP {
```

Comment:
```suggestion
func (n *TraceNotify) OriginalSourceIP() net.IP {
```

Just in case we want to encode the original destination at some point to understand that outbound monitor messages were in fact destined for a service.

(Alternatively we could just call it `ServiceIP` and make sure that the datapath populates it with the service IP, then repurpose the src/dst as pod IPs)

Reply:
I think this is fine for now.